src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -743,3 +743,250 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-tabs]
+== Tabs
+
+The *tabs* page block allows the grouping of different page blocks together.
+Sometimes a page might contain different blocks and simply laying them out side by side might not be the best way to display them.
+Tabs allow you to have the information you need without cluttering the page, making it easier to navigate a page and being more productive.
```

Comment:
I feel like the first and the third sentence say the same thing; can we merge them together?